staging/src/k8s.io/client-go/kubernetes/clientset.go

```diff
@@ -145,17 +154,29 @@ type Clientset struct {
 	coreV1                        *corev1.CoreV1Client
 	eventsV1beta1                 *eventsv1beta1.EventsV1beta1Client
 	extensionsV1beta1             *extensionsv1beta1.ExtensionsV1beta1Client
+	imagepolicyV1alpha1           *imagepolicyv1alpha1.ImagepolicyV1alpha1Client
 	networkingV1                  *networkingv1.NetworkingV1Client
 	policyV1beta1                 *policyv1beta1.PolicyV1beta1Client
 	rbacV1                        *rbacv1.RbacV1Client
-	rbacV1beta1                   *rbacv1beta1.RbacV1beta1Client
 	rbacV1alpha1                  *rbacv1alpha1.RbacV1alpha1Client
+	rbacV1beta1                   *rbacv1beta1.RbacV1beta1Client
 	schedulingV1alpha1            *schedulingv1alpha1.SchedulingV1alpha1Client
 	schedulingV1beta1             *schedulingv1beta1.SchedulingV1beta1Client
 	settingsV1alpha1              *settingsv1alpha1.SettingsV1alpha1Client
-	storageV1beta1                *storagev1beta1.StorageV1beta1Client
 	storageV1                     *storagev1.StorageV1Client
 	storageV1alpha1               *storagev1alpha1.StorageV1alpha1Client
+	storageV1beta1                *storagev1beta1.StorageV1beta1Client
+}
+
+// AdmissionV1beta1 retrieves the AdmissionV1beta1Client
+func (c *Clientset) AdmissionV1beta1() admissionv1beta1.AdmissionV1beta1Interface {
```

Comment:
Why do we have `staging/src/k8s.io/api/admission/v1beta1/types.go` if it's not served? Can we remove it?

Reply:
It's API after all. In this case admission webhooks use it.

Reply:
If I change client-get to skip empty groups, it will skip also `pkg/client/clientset_generated/internalclientset/typed/events/` with it, there is no types.go in `pkg/apis/events/v1beta1` and yet it gets (empty) generated client: https://github.com/kubernetes/kubernetes/blob/f15fe9338484ccfe12c530d1acba7d44ae54e8e9/pkg/client/clientset_generated/internalclientset/typed/events/internalversion/events_client.go#L26-L29

And I don't want to introduce new env. variables to explicitly skip `admission` in update-codegen.sh

Reply:
That empty events client is not used, isn't it? Wouldn't hurt to remove it.